templates/smaily-woocommerce-admin.php

```diff
@@ -356,6 +364,98 @@ class="smaily-toggle"
+						<th scope="row">
+							<label for="checkbox_description">
+							<?php echo esc_html__( 'Subscription checkbox', 'smaily' ); ?>
+							</label>
+						</th>
+						<td id="checkbox_description">
+						<?php
+						esc_html_e(
+							'Customers can subscribe by checking "subscribe to newsletter" checkbox on checkout page.',
+							'smaily'
+						);
+						?>
+						</td>
+					</tr>
+					<tr class="form-field">
+						<th scope="row">
+							<label for="checkbox_display">
+							<?php echo esc_html__( 'Display', 'smaily' ); ?>
+							</label>
+						</th>
+						<td id="smaily_checkout_display_visibility">
+							<?php
+							$checkbox_display_values = array(
+								'visible_unchecked' => __( 'Visible unchecked', 'smaily' ),
+								'visible_checked'   => __( 'Visible checked', 'smaily' ),
+								'hidden_unchecked'  => __( 'Hidden unchecked', 'smaily' ),
+
+							);
+							foreach ( $checkbox_display_values as $dp_value => $dp_translation ) {
+								$checked = $cb_display_selected === $dp_value ? 'checked' : '';
+
+								$input  = '<label for="checkbox_' . $dp_value . '">';
+								$input .= '<input type="radio" ';
+								$input .= 'id="checkbox_' . $dp_value . '" ';
+								$input .= 'name="checkbox_display" ';
+								$input .= 'value="' . $dp_value . '" ' . $checked . '>';
+								$input .= $dp_translation . '</label>';
+								echo( $input );
+							}
+							?>
+						</td>
+					</tr>
+					<tr class="form-field">
+						<th scope="row">
+							<label for="checkbox_order">
+							<?php echo esc_html__( 'Before / After', 'smaily' ); ?>
+							</label>
+						</th>
+						<td>
+							<select id="cb-before-after" name="checkbox_order">
+								<option value="before" <?php echo( 'before' === $cb_order_selected ? 'selected' : '' ); ?> >
+									<?php echo esc_html__( 'Before', 'smaily' ); ?>
+								</option>
+							<option value="after" <?php echo( 'after' === $cb_order_selected ? 'selected' : '' ); ?>>
+								<?php echo esc_html__( 'After', 'smaily' ); ?>
+							</option>
+							</select>
+						</td>
+					</tr>
+					<tr class="form-field">
+						<th scope="row">
+							<label for="checkbox_location">
+							<?php echo esc_html__( 'Location', 'smaily' ); ?>
+							</label>
+						</th>
+						<td>
+							<select id="checkbox-location" name="checkbox_location">
+							<?php
+							$cb_loc_available = array(
+								'order_notes'                => __( 'Order notes', 'smaily' ),
+								'checkout_billing_form'      => __( 'Billing form', 'smaily' ),
+								'checkout_shipping_form'     => __( 'Shipping form', 'smaily' ),
+								'checkout_registration_form' => __( 'Registration form', 'smaily' ),
+							);
+							// Display option and select saved value.
+							foreach ( $cb_loc_available as $loc_value => $loc_translation ) {
+								$selected = $cb_loc_selected === $loc_value ? 'selected' : '';
+								echo( "<option value='$loc_value' $selected>$loc_translation</option>" );
```

Comment:
Same here as with visibility radio buttons.

And please do not mix single and double quotes in HTML (at the moment you have single quotes around options `value` attribute value).

Reply:
Refractored https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/45/commits/524836eba0f93c1cb6b371c34a3bab26635a8385